pkg/securitypolicy/securitypolicyenforcer_rego.go

```diff
@@ -680,3 +683,68 @@ func (policy *regoEnforcer) EnforceScratchUnmountPolicy(scratchPath string) erro
 	}
 	return nil
 }
+
+func getUser(spec *specs.Spec, filter func(user.User) bool) (user.User, error) {
+	users, err := user.ParsePasswdFileFilter(filepath.Join(spec.Root.Path, "/etc/passwd"), filter)
+	if err != nil {
+		return user.User{}, err
+	}
+	if len(users) != 1 {
+		return user.User{}, errors.Errorf("expected exactly 1 user matched '%d'", len(users))
+	}
+	return users[0], nil
+}
+
+func getGroup(spec *specs.Spec, filter func(user.Group) bool) (user.Group, error) {
+	groups, err := user.ParseGroupFileFilter(filepath.Join(spec.Root.Path, "/etc/group"), filter)
+	if err != nil {
+		return user.Group{}, err
+	}
+	if len(groups) != 1 {
+		return user.Group{}, errors.Errorf("expected exactly 1 group matched '%d'", len(groups))
+	}
+	return groups[0], nil
+}
+
+// GetUserInfo returns the user and group information for the container.
```

Comment:
Can you move this comment to the corresponding definition in the `SecurityPolicyEnforcer` interface (ie, `pkg/securitypolicy/securitypolicyenforcer.go` L81)?